build.sbt

```diff
@@ -17,22 +17,27 @@ lazy val scoverageSettings = {
 
 lazy val compile = Seq(
   ws,
-  "uk.gov.hmrc" %% "bootstrap-backend-play-27" % "5.3.0",
-  "uk.gov.hmrc" %% "simple-reactivemongo" % "8.0.0-play-27"
+  "uk.gov.hmrc" %% "bootstrap-backend-play-28" % "5.3.0"
 )
 
+val it = "it"
+
 lazy val test = Seq(
-  "uk.gov.hmrc" %% "hmrctest" % "3.10.0-play-26" % "test",
-  "org.scalatest" %% "scalatest" % "3.0.8" % "test",
-  "com.typesafe.play" %% "play-test" % PlayVersion.current % "test",
-  "org.mockito" % "mockito-all" % "2.0.2-beta" % "test"
+  "uk.gov.hmrc"            %% "bootstrap-test-play-28" % "5.3.0"   % Test,
```

Comment:
updating the `sbt-plugin` necessitated a bunch of additional upgrades, and `mockito` was a bit of a pain.